Utils/utils.py

```
@@ -0,0 +1,216 @@
+import requests
+import re
+import config
+import time
+from typing import Dict
+import json
+from bs4 import BeautifulSoup
+from typing import Any
+import google.generativeai as genai
+from groq import Groq
```

Ruff (0.11.9)

`groq.Groq` imported but unused

Remove unused import: `groq.Groq`

(F401)